# Description: Daemon does not create /var/run/udisks/, so mkfs jobs fail.
# Just create the directory in /tmp, this is what /tmp is for, after all.
# Upstream: https://bugs.freedesktop.org/show_bug.cgi?id=24265

Index: udisks/src/helpers/job-mkfs.c
===================================================================
--- udisks.orig/src/helpers/job-mkfs.c	2010-12-05 23:08:54.587881164 +0100
+++ udisks/src/helpers/job-mkfs.c	2010-12-05 23:09:12.399889312 +0100
@@ -546,7 +546,7 @@
   /* take ownership of the device if requested */
   if (take_ownership_uid != 0 || take_ownership_gid != 0)
     {
-      char dir[256] = PACKAGE_LOCALSTATE_DIR "/run/udisks/job-mkfs-XXXXXX";
+      char dir[256] = "/tmp/job-mkfs-XXXXXX";
 
       if (mkdtemp (dir) == NULL)
         {
